### ISSUE TITLE
Minecolonies citizens entity activation range

### ISSUE THREAD

**Issue by Chrysxv** [Wesley1808/ServerCore#101]
**Describe the bug**
Messing around with the entity activation range feature i'm noticing no matter what I do I can't get minecolonies citizens to be ignored.. Ive tried "minecolonies:citizen, minecolonies:citizens, citizen,  minecolonies:*  but nothing seems to work. 

**Reproduce**
Steps to reproduce the behavior: Download minecolonies and servercore.. enable activation range.. add minecolonies:* to "excluded entity types".

**Expected behavior**
With minecolonies:* being added under "Excluded entity types" i expected it to ignore the citizens.. But it doesn't.

**Versions** -Latest is not a version!
ServerCore: 1.5.1
Minecraft: 1.20.1

**Mod incompatibilities**
minecolonies-1.20.1-1.1.579-BETA

**Comment by Microjett** [comment]
I'm using all the same as OP but a slightly newer minecolonies version.
I'll have to move to some other mod for entity tps management if the whitelist for entity activation is still not working

**Comment by Wesley1808** [comment]
Hi, I believe this issue ended up being discussed and resolved in discord, hence why its so empty here. https://discord.com/channels/998162243852173402/998164692373622808/1242345330763239505

**Comment by Microjett** [comment]
Hi Wesley1808, 
Appreciate the very fast response!, Interesting info, thanks. I'll see if I can figure out how to target just minecolonies entities with the custom activation type.
Regards,

**Comment by Microjett** [comment]
Ok didn't understand that one can use the mod:entity type like this: minecolonies:citizen 
I did try to use a wildcard like this minecolonies:* but was not able to get that to work.
It does look like using a wildcard is what is tripping it up here.

I just tested again with adding 'minecolonies:citizen' to the excluded entity types list and that is working. As is adding 'minecolonies:citizen' to a custom activation type.

If I try to set 'minecolonies:*' to either the custom activation type or the excluded entity types list it does not work.

**Comment by Wesley1808** [comment]
Yeah that's correct, neither field supports wildcards or entity types that do not exist. As matter of fact it would probably cause the config to not actually reload properly since it considers it invalid, which is why I was looking if there were any errors in the logs about it.
